src/App.svelte

```diff
@@ -31,9 +32,17 @@
     }
   }
 
+  function getEpisodeNumberFromQueryParams(): string {
+    const episodeNumber = queryString.parse(location.search).episode;
+    return Array.isArray(episodeNumber) ? episodeNumber[0] : episodeNumber;
```

Comment:
The use of this new package `query-string` seems unmotivated when we're already using `URLSearchParams`. We're parsing the query string twice here.

Would like if you used the [`searchParams.get()`](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/get) here, for example:

```suggestion
    return searchParams.get("episode");
```

And then removing the dependency on `query-string`.

Reply:
You're right! 

On it.

Reply:
Done!